Neos.ContentRepository.BehavioralTests/Tests/Behavior/Features/EventSourced/Migration/AddNewProperty_NoDimensions.feature

```diff
@@ -64,7 +66,7 @@ Feature: Add New Property
           -
             type: 'AddNewProperty'
             settings:
-              newPropertyName: 'aDateOutsideSchema'
+              newPropertyName: 'dateTime'
               serializedValue: '2013-09-09T12:04:12+00:00'
               type: 'DateTime'
```

Comment:
i thought we wanted to remove the obsolete typing here? But might be fine to be explicit ;) Maybe use the node type ones as fallback

Reply:
To deserialize the property the type must be known. And since it's a _new_ property, we can't ask the `$node` instance about it.
So we would have to inject the `NodeTypeManager` bla di bla.
I think it's fine to do it explicitly